hr_timesheet_hours_month/views/project_task_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_project_task_form" model="ir.ui.view">
+        <field name="name">project.task.form</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='planned_hours']" position="after">
+                <field name="planned_hours_year" string="Planned hours to year"/>
+                <field name="remaining_hours_month" string="Remaining hours month" attrs="{'invisible': [('planned_hours_year','=',False)]}"/>
```

Comment:
Use proper widget that formats this value to `hh:mm`, as other float time fields.